lib/src/vector_drawable.dart

```diff
@@ -1153,6 +1163,15 @@ class DrawableShape implements DrawableStyleable {
     } else {
       innerDraw();
     }
+    if (style.mask != null) {
+      final Paint p = Paint();
+      p.blendMode = BlendMode.dstIn;
+      p.colorFilter = ColorFilter.matrix(<double>[0,0,0,0,0, 0,0,0,0,0, 0,0,0,0,0, 0.21,0.71,0.08,0,0]); //convert grayscale values to transparency
```

Comment:
This seems right, but could you provide a link to the spec for this in a comment?